Assets/UXF/Scripts/Etc/Tracker.cs

```diff
@@ -44,7 +45,11 @@ public string DataName
         public bool Recording { get { return recording; } }
```

Comment:
We can make this property return `currentState == TrackerState.On` and remove the `recording` field completely. Calls to `recording` in this class should be replaced with `currentState == TrackerState.On`